shell/models/namespace.js

```diff
@@ -260,4 +260,8 @@ export default class Namespace extends SteveModel {
   get hideDetailLocation() {
     return !!this.$rootGetters['currentProduct'].hideNamespaceLocation;
   }
+
+  get isIstioInjectionEnabled() {
```

Comment:
@torchiaf There already is `injectionEnabled` defined on the model that we can use

Reply:
Updated, Thanks.